### PR TITLE
[CI][VSTS] Add links to the created pkgs.

### PR DESCRIPTION
**Pull request by mandel-macaque**
Add a list with all the pkgs created to make users life better.

fixes: https://github.com/xamarin/xamarin-macios/issues/10298

### CODE REVIEW

File: tools/devops/automation/templates/packages/publish-html.yml [hunk 1]
```diff
@@ -62,16 +70,34 @@ steps:
       Write-Host "##vso[task.setvariable variable=$envVarName]$url"
     }
   displayName: 'Create API from stable diff gists'
+  enabled: false
```
Comment (mandel-macaque) [inline tools/devops/automation/templates/packages/publish-html.yml:73]: The gist is giving me issues with the token, will work on it in a diff PR, but I want the pkgs in the comment asap.

File: tools/devops/automation/templates/packages/publish-html.yml [hunk 2]
```diff
@@ -62,16 +70,34 @@ steps:
...
+    DEVICE_PREFIX: ${{ parameters.devicePrefix }}
+    GITHUB_TOKEN: $(GitHub.Token)
+    TESTS_JOBSTATUS: $(TESTS_JOBSTATUS) # set by the runTests step
+    TESTS_SUMMARY: $(TEST_SUMMARY_PATH)
+    ACCESSTOKEN: $(System.AccessToken)
+  displayName: 'Add summaries'
   condition: always()
   timeoutInMinutes: 1
 
+
```
Comment (tj-devel709) [inline tools/devops/automation/templates/packages/publish-html.yml:100]: Is added spacing here on purpose?

**Review by tj-devel709 — APPROVED** [review]

File: tools/devops/automation/templates/packages/publish-html.yml (hunk 2 above)
Comment (mandel-macaque) [reply tools/devops/automation/templates/packages/publish-html.yml:100]: Not really, I can clean it up, I think is was by diff tool

**Review by dalexsoto — APPROVED** [review]

**Comment by mandel-macaque** [comment]
Fix is know and solved by https://github.com/xamarin/xamarin-macios/pull/10366

**Pull request merged**